core/src/main/java/hudson/util/AtomicFileWriter.java

```diff
@@ -246,11 +247,11 @@ private static void move(Path source, Path destination) throws IOException {
                 // Both files are on the same filesystem, so this should not happen.
                 LOGGER.log(Level.WARNING, e, () -> "Atomic move " + source + " → " + destination + " not supported. Falling back to non-atomic move.");
                 atomicMoveSupported = false;
+            } catch (AccessDeniedException e) {
+                LOGGER.log(Level.INFO, e, () -> "Move " + source + " → " + destination + " failed, perhaps due to a temporary file lock. Falling back to non-atomic move.");
```

Comment:
Can we use here `->` instead of a single character that dennotes the arrow? In the windows console this looks garbled:
`...\build.xml-atomic15623552239278973924tmp ÔåÆ c:\temp\jenkins\...`
Also in line 248

Reply:
I think you neglected to configure your server to use UTF-8?

Reply:
`chcp 65001` in the cmd window before starting Jenkins fixes it for me.

Reply:
Very intuitive! :grin: